website/client/components/guilds/index.vue

```diff
@@ -0,0 +1,67 @@
+<template lang="pug">
+.row
+  .col-12
+    nav.nav.sub-nav
+      .col-md-6.offset-md-4
+        router-link.nav-link(:to="{name: 'tavern'}", exact, :class="{'active': $route.name === 'tavern'}") {{ $t('tavern') }}
+        router-link.nav-link(:to="{name: 'myGuilds'}", :class="{'active': $route.name === 'myGuilds'}") {{ $t('myGuilds') }}
+        router-link.nav-link(:to="{name: 'guildsDiscovery'}", :class="{'active': $route.name === 'guildsDiscovery'}") {{ $t('guildsDiscovery') }}
+      .col-md-2
+        button.btn.btn-primary.btn-purple(b-btn, @click="$root.$emit('show::modal','guild-form')") {{ $t('createGuild') }}
+  .col-12
+    router-view
+
+  group-form-modal
+</template>
+
+<style lang="scss" scoped>
+  body {
+    background-color: #f9f9f9
+  }
+
+  .sub-nav {
```

Comment:
There's a submenu already implemented in components/secondaryMenu that I've used in other views